vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -149,6 +154,11 @@ void AdsImpl::InitializeStep4(
 
   NotificationAllowedCheck(false);
 
+#if defined(OS_ANDROID)
+    RemoveAllNotificationsAfterReboot();
+    RemoveAllNotificationsAfterUpdate();
```

Comment:
Please follow Chromium patterns and append `IfNeeded`

Reply:
NONCRITICAL
Covered by: brave/brave-browser#6235 (6)